dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/claims/nexusmutual_ethereum_claims_v1.sql

```diff
@@ -2,12 +2,13 @@
   config(
     schema = 'nexusmutual_ethereum',
     alias = 'claims_v1',
-    materialized = 'view',
+    materialized = 'table',
+    tags = ['static'],
```

Comment:
```suggestion
```

Reply:
you should remove static, as it reads from tables. we use `static` when the spell is just hardcoded strings and no table reads

Reply:
ah cool, so if all I want is to have this data as a table which is loaded once and just sits there - `materialized = 'table'` is all I need?

Reply:
we have removed usage of `static` tag in our prod orchestration atm. you could actually leave in then if you want, so if ever in the future we want to revisit adding, we would know this model can be static

Reply:
good to know, let's leave it it then